app/upgrades/v10/upgrades_test.go

```diff
@@ -0,0 +1,48 @@
+package v10_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/Stride-Labs/stride/v9/app/apptesting"
+	v10 "github.com/Stride-Labs/stride/v9/app/upgrades/v10"
+)
+
+type UpgradeTestSuite struct {
+	apptesting.AppTestHelper
+}
+
+func (s *UpgradeTestSuite) SetupTest() {
+	s.Setup()
+}
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(UpgradeTestSuite))
+}
+
+func (s *UpgradeTestSuite) TestUpgrade() {
+	dummyUpgradeHeight := int64(5)
+
+	s.ConfirmUpgradeSucceededs("v10", dummyUpgradeHeight)
+
+	// Check mint parameters after upgrade
+	zeroPad := "00000000000000"
```

Comment:
Could you explain this?

Reply:
When converting a decimal to a string, it tacks on the extra zeros, so I'm adding it to the test here for the comparison. The alternative is to use .Equals, but then it doesn't print the value of each so it's hard to debug